databricks/sdk/_base_client.py

```diff
@@ -68,6 +69,8 @@ def __init__(self,
         :param extra_error_customizers:
         :param debug_headers: Whether to include debug headers in the request log.
         :param clock: Clock object to use for time-related operations.
+        :param streaming_buffer_size: The size of the buffer to use for streaming responses. If None, all the
+            response content is loaded into memory at once.
```

Comment:
Is this actually practical to allow reading all into memory? If that's achievable by passing big enough buffer size, do we need to make this parameter optional?

Reply:
My original intent was to provide a way to keep the current behavior. Though, I agree this is something we might want to simply remove for clarity. One can achieve a similar behavior by setting the size to an arbitrarily large number if needed.